.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: /clients/node
+    allow:
+      - dependency-type: development
+    ignore:
+      - dependency-type: production
+      - dependency-name: @types/node
+        versions:
+          - ">=19"
```

Comment:
we want to update packages used in building and testing, but "production" dependencies should really be set by _clients_ of this library.